### PR TITLE
D8 nid 1124

### PR DESCRIPTION
**Pull request by omahm**

### CODE REVIEW

**Review by DuttonMa — APPROVED** [review]
Nice work

File: origins_common/src/Plugin/Filter/CookieContentBlockerEmbedFilter.php [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm(array $form, FormStateInterface $form_state) {
+    $form['replacement_text'] = [
+      '#type' => 'textfield',
+      '#title' => $this->t('Replacement link text'),
+      '#description' => $this->t('Text for the link to the embedded content.'),
+      '#default_value' => $this->settings['replacement_text'] ?? 'Click here to view the video content',
```
Comment (johangant) [inline origins_common/src/Plugin/Filter/CookieContentBlockerEmbedFilter.php:91]: Wrap 'Click here to view the video content' in `$this->t()`) ?

File: origins_common/src/Plugin/Filter/CookieContentBlockerEmbedFilter.php [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+  public function process($text, $langcode) {
+
+    // Quick performance check for drupal media tags within the content.
+    if (stripos($text, '<drupal-media') === FALSE) {
+      return new FilterProcessResult($text);
+    }
+
+    // Ensure the Cookie Content Blocker module is installed.
+    if ($this->moduleHandler->moduleExists('cookie_content_blocker') === FALSE) {
+      $this->logger->error('Cookie Content Blocker Embed Filter is enabled for text formats but the Cookie Content_Blocker is not installed.');
```
Comment (johangant) [inline origins_common/src/Plugin/Filter/CookieContentBlockerEmbedFilter.php:131]: Translate string needed? `$this->t();`

**Review by johangant — APPROVED** [review]
Looks great. I spotted two instances where the text can go through the translation function (usual thing I seem to spot) but otherwise 👍

**Pull request merged**